docs/manuals/gui/viselements/charts/filled-area-plots.md

```diff
@@ -0,0 +1,155 @@
+## Filled Area Plots
+
+An area chart displays a solid color between the traces of a graph. It is an option on scatter charts. Areas will be filled according to the ‘fill’ option ("none", "tozeroy", "tozerox" , "tonexty", "tonextx", "toself", and "tonext")
+
+### Simple example
+This example shows the number of sold items per day (The shop closes on Weekend :) ).
```

Comment:
=>
"""
The simplest example is where you have a single dataset to represent.

Here is how we can represent the number of items sold every weekday:
"""